000-configure-stats.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+
+# make bash behave
+set -euo pipefail
+IFS=$'\n\t'
+
+if [ -n "${DISABLE_STATS_COLLECTION+set}" ]; then
```

Comment:
One minor thought. This docker var is written in the negative tone, "DISABLE_..." while the guc is in the affirmative, "enable_..." What if the docker var was `ENABLE_STATS_COLLECTION` and defaulted to true, but could be set to false?

I don't feel strongly about it, just thought I'd point it out.

Reply:
I kind of liked having it just be "is this variable set" rather than checking for `true`/`false`. In particular, people might set it to `True` or `1` or `On` or whatever and then be confused. Lots of projects use "is this set" as the semantics.

In addition, Citus already has some other GUCs that start with `enable_` so that's why we went that direction there. I know this means the two uses are opposite, but I don't think it'll result in too much confusion, I hope.